pkg/network/port_test.go

```diff
@@ -120,17 +121,15 @@ func TestReadInitialUDPState(t *testing.T) {
 	err := exec.Command("testdata/setup_netns.sh", nsName).Run()
 	require.NoError(t, err, "setup_netns.sh failed")
 
-	l, err := net.ListenUDP("udp", &net.UDPAddr{})
-	require.NoError(t, err)
-	defer func() { _ = l.Close() }()
+	l := nettestutil.StartServerUDP(t, net.ParseIP("0.0.0.0"), 5432)
+	defer l.Close()
```

Comment:
nit: prefer t.Cleanup() https://stackoverflow.com/questions/61609085/what-is-useful-for-t-cleanup